addons/website/views/website_templates.xml

```diff
@@ -918,18 +918,19 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
     </url>
 </template>
 
-<template id="sitemap_xml">&lt;?xml version="1.0" encoding="UTF-8"?&gt;
+<template id="sitemap_xml"><t t-translation="off">&lt;?xml version="1.0" encoding="UTF-8"?&gt;</t>
```

Comment:
Why this one `UTF-8` is in upper and the other in lower? Should be homogenized now that you are touching?

Reply:
No specific reason, is there one is better?

Reply:
In upper seems a good decision :)

https://en.wikipedia.org/wiki/XML#Encoding_detection